packages/math-input/src/components/many-keypad-button.tsx

```diff
@@ -15,11 +15,12 @@ import TouchableKeypadButton from "./touchable-keypad-button";
 import type {Key} from "../data/keys";
 
 type Props = {
-    keys?: Array<Key>;
+    keys?: ReadonlyArray<Key>;
 };
 
 class ManyKeypadButton extends React.Component<Props> {
     render() {
+        // do we even need the ...rest?
```

Comment:
Don't we use it on line 33 and 40?

Reply:
Yes, but TouchableKeypadButton is kind of a mess and it's a little difficult for me to parse what it actually needs. https://khanacademy.atlassian.net/browse/LC-746

Reply:
Removed